PaddleLite-armlinux-demo/object_detection_demo/object_detection_demo.cc

```diff
@@ -144,100 +146,164 @@ std::vector<RESULT> postprocess(const float *output_data, int64_t output_size,
-         WARMUP_COUNT, REPEAT_COUNT, total_time_cost / REPEAT_COUNT,
-         max_time_cost, min_time_cost);
 
-  // 5. Get the data of output tensor and postprocess to output detected objects
+  // Get the data of output tensor and postprocess to output detected objects
   std::unique_ptr<const paddle::lite_api::Tensor> output_tensor(
       std::move(predictor->GetOutput(0)));
   const float *output_data = output_tensor->mutable_data<float>();
   int64_t output_size = 1;
-  for (auto dim : output_tensor->shape())
+  for (auto dim : output_tensor->shape()) {
     output_size *= dim;
+  }
   cv::Mat output_image = input_image.clone();
+  double postprocess_start_time = get_current_us();
   std::vector<RESULT> results = postprocess(
-      output_data, output_size, word_labels, SCORE_THRESHOLD, output_image);
-  cv::imwrite(output_image_path, output_image);
-  // imshow("object detection demo", output_image);
-  printf("results: %d\n", results.size());
-  for (int i = 0; i < results.size(); i++) {
-    printf("[%d] %s - %f %f,%f,%f,%f\n", i, results[i].class_name.c_str(),
-           results[i].score, results[i].left, results[i].top, results[i].right,
-           results[i].bottom);
+      output_data, output_size, word_labels, SCORE_THRESHOLD, output_image, prediction_time);
+  double postprocess_end_time = get_current_us();
+  double postprocess_time = (postprocess_end_time - postprocess_start_time) / 1000.0f;
+  printf("Postprocess time: %f ms\n\n", postprocess_time);
+
+  if (!enable_camera) {
+    printf("results: %d\n", results.size());
+    for (int i = 0; i < results.size(); i++) {
+      printf("[%d] %s - %f %f,%f,%f,%f\n", i, results[i].class_name.c_str(),
+             results[i].score, results[i].left, results[i].top, results[i].right,
+             results[i].bottom);
+    }
+  }
+  return output_image;
+}
+
+int main(int argc, char **argv) {
+  if (argc < 3 || argc == 4) {
+    printf(
+        "Usage: \nIf you want to use video to do pretict, please set"
+        "\n\tobject_detection_demo model_dir label_path\n"
+        "\nIf you want to use image to do pretict, please set"
+        "\n\tobject_detection_demo model_dir label_path input_image_path "
+        "output_image_path\n");
+    return -1;
+  }
+
+  std::string model_dir = argv[1];
+  std::string label_path = argv[2];
+  std::string input_image_path;
+  std::string output_image_path;
+  bool enable_camera = true;
+
+  if (argc > 3) {
+    enable_camera = false;
+    input_image_path = argv[3];
+    output_image_path = argv[4];
+  }
+
+  std::vector<std::string> word_labels = load_labels(label_path);
+
+  paddle::lite_api::MobileConfig config;
+  config.set_model_dir(model_dir);
+  config.set_threads(CPU_THREAD_NUM);
+  config.set_power_mode(CPU_POWER_MODE);
+
+  std::shared_ptr<paddle::lite_api::PaddlePredictor> predictor =
+      paddle::lite_api::CreatePaddlePredictor<paddle::lite_api::MobileConfig>(config);
+
+  if (enable_camera) {
```

Comment:
if (argc > 3) {
  // enable camera is input&output image path is not set
  WARMUP_COUNT = 0;
  REPEAT_COUNT = 1;
} else {
  std::string input_image_path = argv[3];
  std::string output_image_path = argv[4];
  cv::Mat input_image = cv::imread(input_image_path, 1);
  cv::Mat output_image = detection(enable_camera, input_image, word_labels, predictor);
  cv::imwrite( output_image_path, output_image);
  cv::imshow("object detection demo", output_image);
  cv::waitKey(0);
}